### DISCUSSION TITLE
It's hard to tell which tab is active in the light color scheme

### DISCUSSION THREAD

**Discussion by Pablo-fdez** [minbrowser/min#1886] [Ideas]
Basically the title. In dark mode the active tab is a bit lighter than the rest. In light mode they are all a very similar color:

![image](https://user-images.githubusercontent.com/58422321/150800997-44f1a494-e0c8-4c90-868b-d6b8e37df251.png)
![image](https://user-images.githubusercontent.com/58422321/150801066-224ba55f-c4bd-47a2-a2f5-c8a26bd42b94.png)

**Comment by ghost** [comment]
Faced a similar problem. But, after a while of usage, I could determine the difference.

**Reply by Pablo-fdez** [reply]
Yes, actually now I think it might have something to do with my brightness settings.

**Comment by strindhaug** [comment]
Yes the contrast is way too low to be accessible. Also the fact that the tab bar changes color depending on the website (manifest file, I assume?) makes it even harder to see the minute difference in color of the faintly highlighted tab because "everything" changes.
Ideally you should be able to set the highlight color yourself, and disable the changing of tab-bar color.

**Reply by strindhaug** [reply]
Just realised, I can disable site themes and choose always night theme. Though I'd still prefer to set the tab bar color and highlight color to something I can see, without also enabling dark mode on all websites that support it.

**Comment by ioc2e3** [comment]
> Basically the title. In dark mode the active tab is a bit lighter than the rest. In light mode they are all a very similar color:

Same opinion ...

.
